Client/Frontend/Menu/AppMenuConfiguration.swift

```diff
@@ -112,7 +116,11 @@ struct AppMenuConfiguration: MenuConfiguration {
             if #available(iOS 9, *) {
                 menuItems.append(AppMenuConfiguration.NewPrivateTabMenuItem)
             }
-            menuItems.append(tabState.isBookmarked ? AppMenuConfiguration.RemoveBookmarkMenuItem : AppMenuConfiguration.AddBookmarkMenuItem)
+            var menuItem = tabState.isBookmarked ? AppMenuConfiguration.RemoveBookmarkMenuItem : AppMenuConfiguration.AddBookmarkMenuItem
+            if let errorPageUrl = tabState.url where ErrorPageHelper.isErrorPageURL(errorPageUrl) {
```

Comment:
includeDataURIs: false results in a compiler error: "Argument passed to call that takes no arguments". Looks like the function no longer takes arguments. 

I assume includeDataURIs: false makes us exclude reader mode pages which are probably stored locally as data-uris. Is that correct? How come we still display the actual http url in the address bar though?